src/ppom.js

```diff
@@ -0,0 +1,433 @@
+      };
+      const ret = new Promise(cb0);
+      return addHeapObject(ret);
+    } finally {
+      state0.a = state0.b = 0;
+    }
+  };
+  imports.wbg.__wbg_resolve_fd40f858d9db1a04 = function (arg0) {
+    const ret = Promise.resolve(getObject(arg0));
+    return addHeapObject(ret);
+  };
+  imports.wbg.__wbg_then_ec5db6d509eb475f = function (arg0, arg1) {
+    const ret = getObject(arg0).then(getObject(arg1));
+    return addHeapObject(ret);
+  };
+  imports.wbg.__wbg_then_f753623316e2873a = function (arg0, arg1, arg2) {
+    const ret = getObject(arg0).then(getObject(arg1), getObject(arg2));
+    return addHeapObject(ret);
+  };
+  imports.wbg.__wbindgen_string_get = function (arg0, arg1) {
+    const obj = getObject(arg1);
+    const ret = typeof obj === 'string' ? obj : undefined;
+    var ptr0 = isLikeNone(ret)
+      ? 0
+      : passStringToWasm0(ret, wasm.__wbindgen_malloc, wasm.__wbindgen_realloc);
+    var len0 = WASM_VECTOR_LEN;
+    getInt32Memory0()[arg0 / 4 + 1] = len0;
+    getInt32Memory0()[arg0 / 4 + 0] = ptr0;
+  };
+  imports.wbg.__wbg_parse_3ac95b51fc312db8 = function () {
+    return handleError(function (arg0, arg1) {
+      const ret = JSON.parse(getStringFromWasm0(arg0, arg1));
+      return addHeapObject(ret);
+    }, arguments);
+  };
+  imports.wbg.__wbg_stringify_029a979dfb73aa17 = function () {
+    return handleError(function (arg0) {
+      const ret = JSON.stringify(getObject(arg0));
+      return addHeapObject(ret);
+    }, arguments);
+  };
+  imports.wbg.__wbindgen_throw = function (arg0, arg1) {
+    throw new Error(getStringFromWasm0(arg0, arg1));
+  };
+  imports.wbg.__wbindgen_closure_wrapper_wasm_bindgen__closure__Closure_T___wrap__breaks_if_inlined =
+    function (arg0, arg1, arg2) {
+      const ret = makeMutClosure(
+        arg0,
+        arg1,
+        __wbg_adapter_13,
+        __wbg_adapter_12,
+      );
+      return addHeapObject(ret);
+    };
+
+  return imports;
+}
+
+function initMemory(imports, maybe_memory) {}
+
+function finalizeInit(instance, module) {
+  wasm = instance.exports;
+  init.__wbindgen_wasm_module = module;
+  cachedInt32Memory0 = null;
+  cachedUint8Memory0 = null;
+
+  wasm.__wbindgen_start();
+  return wasm;
+}
+
+function initSync(module) {
+  const imports = getImports();
+
+  initMemory(imports);
+
+  if (!(module instanceof WebAssembly.Module)) {
+    module = new WebAssembly.Module(module);
+  }
+
+  const instance = new WebAssembly.Instance(module, imports);
+
+  return finalizeInit(instance, module);
+}
+
+async function init(input) {
+  const imports = getImports();
+
+  if (
+    typeof input === 'string' ||
+    (typeof Request === 'function' && input instanceof Request) ||
+    (typeof URL === 'function' && input instanceof URL)
+  ) {
+    input = fetch(input);
+  }
+
+  initMemory(imports);
+
+  const { instance, module } = await load(await input, imports);
+
+  return finalizeInit(instance, module);
+}
+
+export { initSync };
+export default init;
```

Comment:
This code is bit of blackbox for me, it has been shared by Blockaid's team. I am not sure what will be best way to include it in our code

I excluded it from lint and tests.

Reply:
Blockaid team shared that this is an file autogenerated for wasm code and should not be updated manually. I added this information in a comment in the file.